docs/installation.rst

```diff
@@ -63,6 +63,10 @@ libspotify in the homebrew/binary tap::
 
     brew install homebrew/binary/libspotify
 
+Install libffi as a dependency::
+
+    brew install libffi
+
```

Comment:
Fair enough, I'm very sorry for the hasty pull request. I'll create an issue later instead. `libffi` is indeed provided by OS X, I had some weird error and don't have the logs at hand. Sorry for the inconvenience.